NEMP/mods.json

```diff
@@ -4561,5 +4561,14 @@
             "promo": "recommended",
             "promoType": "version"
         }
+    },
+    "Brain Stone Mod": {
+        "function" : "CheckCurse",
+        "active" : true,
+        "curse" : {
+            "id": "250836",
+            "name": "brainstonemod",
```

Comment:
This is the slug part from the [Curse project page URL](https://mods.curse.com/mc-mods/minecraft/250836-brain-stone-mod), so it should be changed to `brain-stone-mod`. The ID portion is correct.